CMakeLists.txt

```diff
@@ -520,27 +520,16 @@ target_compile_definitions(core PRIVATE
     ${GIT_BUILD_INFO_DEFINES}
 )
 
+target_link_libraries(core
+    OpenMP::OpenMP_CXX
+    poly2tri
+    tinyxml
+    visilibity
```

Comment:
I thought `visilibity` is not used anymore.

Reply:
I just did a quick check, `routing/smoke_router/cognitiveMap/cognitivemap.cpp` still uses it